.github/workflows/major-version-upgrade.yml

```diff
@@ -189,14 +189,27 @@ jobs:
           } > pg_hba.conf
         shell: bash
 
+      - name: Check Babelfish metadata inconsistency before Major Version Upgrade
+        id: check-babelfish-inconsistency
+        if: always() && steps.setup-new-datadir.outcome == 'success'
+        run: |
+          output=$(sqlcmd -S localhost -U jdbc_user -P 12345678 -Q "SELECT sys.check_for_inconsistent_metadata() GO" | tail -n +2)
+          echo "check_result=$output" >> "$GITHUB_OUTPUT"
```

Comment:
Might make sense to move this logic into a composite action and then call that composite action in all the WFs